app/assets/config/manifest.js

```diff
@@ -0,0 +1,3 @@
+//= link_tree ../images
```

Comment:
Is this file required?

Reply:
It is needed by the Sprockets gem (Rails asset pipeline is implemented by this gem).